index.html

```diff
@@ -0,0 +1,228 @@
+                    <a href="#"><i class="fa fa-facebook"></i></a>
+                </li>
+                <li class="col-md-1">
+                    <a href="# "><i class="fa fa-twitter"></i></a>
+                </li>
+                <li class="col-md-1">English</li>
+                <li class="col-md-1">My Page</li>
+                <li class="col-md-1">Logoout</li>
+                <li class="col-md-1"></li>
+            </ul>
+        </div>
+        <div class="container-fluid header2">
+            <i class="mt-5 hamburger-menu openMenu fa fa-bars"></i>
+            <ul class="row header2List mainMenu">
+                <li class="closeMenu"><i class="closeMenu fa fa-times"></i></li>
+                <li class="d-none d-md-block col-md-3 p-3">
+                    <a href="#" class="logo"><img src="./image/logo.png" alt="women image logo" />
+                    </a>
+                </li>
+                <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+                    <a class="nav-link" href="./about.html">
+              About<span class="navSpanArrow1">></span>
+            </a>
+                </li>
+                <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+                    <a class="nav-link" href="#programs">
+              Program<span class="navSpanArrow2">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+            <a class="nav-link" href="#speakers">
+              Join<span class="navSpanArrow3">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+            <a class="nav-link" href="#sponsor">
+              Sponsor<span class="navSpanArrow2">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-1 px-1 py-3 nav-item">
+            <a class="nav-link" href="#">
+              News<span class="navSpanArrow4">></span></a
+            >
+          </li>
+          <li class="d-md-block col-md-2 py-3 EAWDicore nav-item">
+            <a class="nav-link" href="#AWECampaine">
+              EAW Campaign<span class="navSpanArrow5">></span></a
+            >
+          </li>
+          <li class="col-md-1"></li>
+        </ul>
+      </div>
+    </header>
+    <section class="main">
+      <div class="headerTitle container" id="AWECampaine">
+        <h2 class="headerTitleh2 mb-2">"HELLOW! AFRICAN WOMEN"</h2>
+        <div class="redBG">
+          <h1 class="headerTitleh1 mb-1">
+            EMPOWERING AFRICAN WOMEN <br />
+            GLOBAL SUMMIT <br />
+            2022
+          </h1>
+        </div>
+
+        <p class="headerTitleP">
+          A joyful celebration believing in the value of empowering women to
+          achieve their destiny by removing barriers that hold them back with
+          people from over 54 countries is taking place in August, Ethiopia.
+        </p>
+        <div class="headerDate">2022.08.11(THU) ~ 12(FRI)</div>
```

Comment:
- [ ] Kindly make the date under this section a bit bolder, as can be seen in the [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).